src/include/common/vector/value_vector_utils.h

```diff
@@ -7,10 +7,6 @@ namespace common {
 
 class ValueVectorUtils {
 public:
-    // These two functions assume that the given uint8_t* srcData/dstData are pointing to a data
-    // with the same data type as this ValueVector.
-    static void copyNonNullDataWithSameTypeIntoPos(
-        ValueVector& resultVector, uint64_t pos, const uint8_t* srcData);
     static void copyNonNullDataWithSameTypeOutFromPos(const ValueVector& srcVector, uint64_t pos,
```

Comment:
Maybe we can move copyNonNullDataWithSameTypeOutFromPos to factorizedTable following the same style?